plugins/experimental/webp_transform/ImageTransform.cc

```diff
@@ -183,7 +183,7 @@ class GlobalHookPlugin : public GlobalPlugin
 
     // If we might need to convert check to see if what the browser supports
     if (transaction_convert_to_webp == true || transaction_convert_to_jpeg == true) {
-      std::string accept  = transaction.getServerRequest().getHeaders().values("Accept");
+      std::string accept  = transaction.getClientRequest().getHeaders().values("Accept");
       bool webp_supported = accept.find("image/webp") != std::string::npos;
```

Comment:
Seems wrong that we just assume the client will accept "image/jpeg" regardless of what's in the accept header the client sent.

Reply:
If something modifies Accept header, the something should be responsible for returning a content that the client accept.